### PR TITLE
Feature/add sharing sidebar

### PR DESCRIPTION
**Pull request by DeepDiver1975**
followup of #913

### CODE REVIEW

File: apps/files/src/components/FileSharingSidebar.vue [hunk 1]
```diff
@@ -1,13 +1,206 @@
 <template>
-    <div>
-      <p>file sharing sidebar</p>
+  <div>
+    <div v-if="selectedFiles.length > 1">
+      <span v-translate>Please choose only a single File</span>
+    </div>
+    <div v-else>
+     <oc-spinner v-if="sharesLoading"></oc-spinner>
+    <oc-accordion v-if="!sharesLoading" uk-accordion="collapsible: false; multiple: true">
```
Comment (LukasHirt) [inline apps/files/src/components/FileSharingSidebar.vue]: If I collapse one item I can't collapse the other one. Is it on purpose like this? If `collapsible: false;` is removed then the problem is gone.

Comment (DeepDiver1975) [reply apps/files/src/components/FileSharingSidebar.vue]: no idea if intended .... feel free to change an push to this branch

**Comment by ownclouders** [comment]
Find screenshots at https://minio.owncloud.com/phoenix/screenshots/2670/

**Comment by ownclouders** [comment]
Find screenshots at https://minio.owncloud.com/phoenix/screenshots/2672/

**Comment by DeepDiver1975** [comment]
```
 Element <#new-file-menu-btn> was visible after 31 milliseconds.
 Error while running .clickElement() protocol action: element click intercepted: Element <button class="oc-button uk-button-primary" id="new-file-menu-btn" aria-expanded="false">...</button> is not clickable at point (813, 90). Other element would receive the click: <div uk-dialog="{bgclose: false, escclose: false}" class="uk-modal-dialog uk-modal-body">...</div>

    ✖ failed
      Error while running "click" command: this.reporter.registerTestError is not a function
      TypeError: Error while running "click" command: this.reporter.registerTestError is not a function
          at transport.locateElement.then.then.catch.result (/var/www/owncloud/phoenix/node_modules/nightwatch/lib/api-loader/element-command.js:138:25)
          at <anonymous>
          at process._tickCallback (internal/process/next_tick.js:189:7)
    And the user deletes folder "my-empty-folder" using the webUI
    - skipped
    Then as "user1" folder "my-other-empty-folder" should exist
    - skipped
    And folder "my-other-empty-folder" should be listed on the webUI
    - skipped
    But as "user1" folder "my-empty-folder" should not exist
    - skipped
    And folder "my-empty-folder" should not be listed on the webUI
    - skipped

**Pull request merged**